app/assets/javascripts/multiprogress.js

```diff
@@ -27,7 +27,15 @@
 			self.progressRoot = self._createProgressRoot();
 			self.progressBottomText = self._createProgressBottomText();
 			self.progressHoverBar = self._createProgressHoverBar();
+
+			// TODO: remove
+      console.log("### multiprogress.js > _create() > before 'self.progressRoot.progressbar'");
+
 			self.progressRoot.progressbar({value: 0, disabled: self.options.disabled}); // Creates one part with width 0%
```

Comment:
@rootux So yes, I've made some progress, but still having a JS error on the progress bar. At this line.

The error is displayed in the JS console when accessing http://dreams.theborderland.se/dreams. Maybe you can help there?

Reply:
@rootux Ok I get it: to get rid of the JS errors I removed jQuery UI, but `progressbar()` is a jQuery UI function. I have an idea...